dev-requirements.txt

```diff
@@ -14,3 +14,4 @@ pydata_sphinx_theme==0.7.1
 nlp_primitives>=2.0.0
 autonormalize >= 1.0.2
 -r koalas-requirements.txt
+holidays>=0.12
```

Comment:
You are going to want this to be in `requirements.txt` as most users wouldn't install the requirements in `dev-requirements.txt` - those are primarily for developers contributing to the codebase.

Reply:
yes, of course. It should be in both place tho, correct?

Reply:
@dvreed77 This requirement only needs to be in one place. The typical path for installing requirements for development would be to run `make installdeps`. After upgrading pip, that make command then installs featuretools in editable mode, and that process will grab everything from `requirements.txt`. Finally, the requirements from `dev-requirements.txt` are installed by the make command, which also includes references to the `test-requirements.txt` and `koalas-requirements.txt` files, so essentially `make installdeps` installs everything found in all of our requirements files.

Reply:
@thehomebrewnerd should I update `contributing.md` to run `make installdeps`? Guide currently has this:

```bash
  virtualenv venv
  source venv/bin/activate
  python -m pip install -e .
  python -m pip install -r dev-requirements.txt
  ```

Reply:
Yeah, I think we should replace the `python*` commands with `make installdeps`. The only real difference is that `make installdeps` will also make sure pip is upgraded first, which should be a good thing to have first.

Reply:
@thehomebrewnerd I added `holiday` package back to dev-requirements because it looks like github actions uses this for install before running unit tests

Reply:
I still think this should be removed from dev requirements and should be covered by the other requirements installs.